pkg/resourcemanager/webhook/nodeagentauthorizer/add.go

```diff
@@ -0,0 +1,30 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package nodeagentauthorizer
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+
+	authorizerwebhook "github.com/gardener/gardener/pkg/utils/webhook/authorizer"
+)
+
+const (
+	// HandlerName is the name of this authorization webhook handler.
+	HandlerName = "nodeagentauthorizer"
```

Comment:
You could document the authorizer webhook in https://github.com/gardener/gardener/blob/450529c84b99ddc36a3ec43abb1e530839fc18a5/docs/concepts/resource-manager.md#L943 or in general the NodeAgentAuthorizer flow can be documented in the node-agent docs similar to the docs we have for the SeedAuthorizer https://github.com/gardener/gardener/blob/3ee1b8ad133df44f28dfd3266e1438ec01da577c/docs/deployment/gardenlet_api_access.md#L133